File: aoj/volume0/n0075/Main.java
import java.util.Scanner;

public class Main {

    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);

        while (sc.hasNext()) {

            String[] input = sc.next().split(",");

            int id = Integer.parseInt(input[0]);
            double weight = Double.parseDouble(input[1]);
            double height = Double.parseDouble(input[2]);
            double bmi = weight / (height * height);

            if (bmi >= 25) {
                System.out.println(id);
            }

        }

    }

}

File: aoj/volume0/n0075/Main.cpp
#include <iostream>

using namespace std;

int main() {

    int id;
    char comma;
    double weight, height;

    while (cin >> id >> comma >> weight >> comma >> height) {

        double ibm = weight / (height * height);

        if (ibm >= 25.0) {
            cout << id << endl;
        }

    }

    return 0;

}
